<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QPushButton" name="logoutBtn">
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startTrainingBtn">
       <property name="text">
        <string>Start training</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startCalibrationBtn">
       <property name="text">
        <string>Start calibration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopTrainingBtn">
       <property name="text">
        <string>Stop training</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resultsBtn">
       <property name="text">
        <string>Results</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
